server.js

```diff
@@ -1,29 +1,119 @@
+/* eslint-disable no-console */
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import listEndpoints from 'express-list-endpoints'
+import dotenv from 'dotenv'
+
+dotenv.config()
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+// eslint-disable-next-line max-len
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const thougthSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: [true, 'Empty message not allowed'],
+    trim: true,
+    minlength: [5, 'Message must be longer than 5 characters'],
+    maxlength: [140, 'Message must be shorter than 140 characters']
```

Comment:
Great that you are sending these messages between line 18-21 as additional information, as well as trimming whitespaces!